<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1064</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabMain">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSnapshots">
       <attribute name="title">
        <string>Snapshots</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTreeWidget" name="twSnapshots">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::ActionsContextMenu</enum>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideNone</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <attribute name="headerShowSortIndicator" stdset="0">
            <bool>true</bool>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Timestamp</string>
            </property>
            <property name="textAlignment">
             <set>AlignRight|AlignVCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Heap size (KiB)</string>
            </property>
            <property name="textAlignment">
             <set>AlignRight|AlignVCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Heap extra size (KiB)</string>
            </property>
            <property name="textAlignment">
             <set>AlignRight|AlignVCenter</set>
            </property>
           </column>
          </widget>
          <widget class="QSplitter" name="splitter">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>1</horstretch>
             <verstretch>6</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="HistoryGraph" name="grHistory" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>5</verstretch>
             </sizepolicy>
            </property>
           </widget>
           <widget class="QTreeView" name="tvHistory">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="uniformRowHeights">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCodeLocations">
       <attribute name="title">
        <string>Code locations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="tvCodeLocations">
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="headerStretchLastSection">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1064</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actProjectNew"/>
    <addaction name="actProjectOpen"/>
    <addaction name="actProjectSave"/>
    <addaction name="actProjectSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actProjectExit"/>
   </widget>
   <widget class="QMenu" name="menu_Snapshots">
    <property name="title">
     <string>&amp;Snapshots</string>
    </property>
    <addaction name="actSnapshotsAdd"/>
    <addaction name="separator"/>
    <addaction name="actSnapshotsExportSelected"/>
    <addaction name="actSnapshotsExportAll"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Snapshots"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actProjectNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actProjectOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actProjectExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actSnapshotsAdd">
   <property name="text">
    <string>&amp;Add from file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actSnapshotsExportSelected">
   <property name="text">
    <string>E&amp;xport selected...</string>
   </property>
  </action>
  <action name="actSnapshotsExportAll">
   <property name="text">
    <string>&amp;Export all...</string>
   </property>
  </action>
  <action name="actCtxDiffSelected">
   <property name="text">
    <string>Diff selected...</string>
   </property>
  </action>
  <action name="actCtxDiffAll">
   <property name="text">
    <string>Diff all</string>
   </property>
  </action>
  <action name="actProjectSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actProjectSaveAs">
   <property name="text">
    <string>Save &amp;as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>HistoryGraph</class>
   <extends>QWidget</extends>
   <header>HistoryGraph.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actProjectExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
